lstmcpipe/stages/mc_r0_to_dl1.py

```diff
@@ -213,69 +300,62 @@ def r0_to_dl1(input_dir, config_file=None, train_test_ratio=0.5, random_seed=42,
         check_and_make_dir_without_verification(dir_lists)
         check_and_make_dir_without_verification(output_dir)
 
-        print("\toutput dir: \t", output_dir)
+        log.info("output dir: {}".format(output_dir))
 
-        number_of_sublists = len(list_type) // N_R0_PER_DL1_JOB + int(len(list_type) % N_R0_PER_DL1_JOB > 0)
+        number_of_sublists = len(list_type) // n_r0_per_dl1_job + int(len(list_type) % n_r0_per_dl1_job > 0)
         for i in range(number_of_sublists):
             output_file = os.path.join(dir_lists, '{}_{}.list'.format(set_type, i))
             with open(output_file, 'w+') as out:
-                for line in list_type[i * N_R0_PER_DL1_JOB:N_R0_PER_DL1_JOB * (i + 1)]:
+                for line in list_type[i * n_r0_per_dl1_job:n_r0_per_dl1_job * (i + 1)]:
                     out.write(line)
                     out.write('\n')
-        print(f'\t{number_of_sublists} files generated for {set_type} list')
+        log.info(f'{number_of_sublists} files generated for {set_type} list')
 
-        # LSTCHAIN #
-        counter = 0
         save_job_ids = []
 
-        for file in os.listdir(dir_lists):
-            if set_type == 'training':
-                jobo = os.path.join(JOB_LOGS, f"job{counter}_train.o")
-                jobe = os.path.join(JOB_LOGS, f"job{counter}_train.e")
-            else:
-                jobo = os.path.join(JOB_LOGS, f"job{counter}_test.o")
-                jobe = os.path.join(JOB_LOGS, f"job{counter}_test.e")
-
-            job_name = {'electron': f'e_{jobtype_id}_r0dl1',
-                        'gamma': f'g_{jobtype_id}_r0dl1',
-                        'gamma-diffuse': f'gd_{jobtype_id}_r0dl1',
-                        'proton': f'p_{jobtype_id}_r0dl1',
-                        'gamma_off0.0deg': f'g0.0_{jobtype_id}_r0dl1',
-                        'gamma_off0.4deg': f'g0.4_{jobtype_id}_r0dl1'
-                        }
-
-            if particle == 'proton':
-                queue = 'long'
-            else:
-                queue = 'short'
+        files = [f.resolve().as_posix() for f in Path(dir_lists).glob("*")]
 
-            cmd = f'sbatch --parsable -p {queue} -J {job_name[particle]} -e {jobe} -o {jobo} ' \
-                  f'--wrap="{base_cmd} -f {os.path.join(dir_lists, file)} -o {output_dir}"'
+        if set_type == 'training':
+            jobo = os.path.join(JOB_LOGS, "job_%A_%a_train.o")
```

Comment:
Are the `%A` and `%a` strings for a later formatting ? I think they are not formatted when passed to the final `cmd`

Reply:
I mean, the final slurm command looks like this :
```
sbatch --parsable --array=[0-0]%20 -p short 
-e {...}/20210907_v0.7.3_prod5_trans_80_test_job_arrays_logging/off0.0deg/job_logs/job_%A_%a_train.e 
-o {...}/20210907_v0.7.3_prod5_trans_80_test_job_arrays_logging/off0.0deg/job_logs/job_%A_%a_train.o 
-J g0.0_LST_r0dl1  --wrap="  ... "
```

Reply:
They are supposed to be filled by the slurm task with the array and task id.
See:
https://slurm.schedmd.com/job_array.html

Reply:
Wow, cool ! 😄